components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml

```diff
@@ -3385,6 +3385,126 @@ paths:
           schema:
             $ref: '#/definitions/Error'
 
+######################################################
+# The "ExternalStore Collection" resource APIs
+######################################################
+  /apis/{apiId}/external-stores:
+
+#-------------------------------------------------------------
+# Retrieve the published external stores list of a certain API
+#-------------------------------------------------------------
+    get:
+      security:
+        - OAuth2Security:
+            - apim:api_view
+      x-wso2-curl: "curl -k -H \"Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8\" \"https://127.0.0.1:9443/api/am/publisher/v1.0/apis/890a4f4d-09eb-4877-a323-57f6ce2ed79b/external-stores\""
+      x-wso2-request: |
+        GET https://127.0.0.1:9443/api/am/publisher/v1.0/apis/890a4f4d-09eb-4877-a323-57f6ce2ed79b/external-stores
+        Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8
+      x-wso2-response: "HTTP/1.1 200 OK\nContent-Type: application/json\n{\"count\":1,\"list\":[{\"id\":\"StoreABC\",\"displayName\":\"StoreABC\",\"type\":\"wso2\"}]}"
+      summary: Get the list of external stores which an API is published to
```

Comment:
Shall we use operationId: getAllExternalStores

This will be the generated method name in both java server code and js client. This will make it easier when calling from swagger js client and also give a good name in the impl classes instead of apisApiidExternalStoresGet

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/6945/commits/d39e99426870187af43b219a4c059199f515fdb7